content/building-blocks/automate-and-execute-your-work/automate-your-workflow/stata-error-handling-make.md

```diff
@@ -70,3 +80,39 @@ target_file: prerequisite.do # define your target and prerequisites
 ```
 
 {{% /codeblock %}}
+
+{{% example %}}
+Imagine you're working on a research project where you have raw data named `raw_data.dta` that needs to be cleaned and processed. You've written a Stata do-file called `data_cleaning.do` that takes this raw data and outputs a cleaned dataset named `clean_data.dta`. 
+
+Additionally, every time you run the `do-file`, a log file named `data_cleaning.log` is generated.
+
+Here's how you can set up your `makefile`:
+```
+clean_data.dta: raw_data.dta data_cleaning.do
+	rm -f data_cleaning.log  
+	StataMP-64.exe -e do data_cleaning.do  
+	Rscript logcheck.R data_cleaning.log  
+
+```
+To use this `makefile`, you would:
+
+1. Place the `makefile` in the directory with your raw data and Stata `do-file`.
+2. Run the `make` command in your terminal or command line. 
+
+This would trigger the data cleaning process using Stata, followed by the log check using R.
+If the R script finds an error in the Stata log, it would interrupt the process and notify you of the issue. If no errors are found, the process completes, and you have your cleaned data ready for analysis!
+
+<p align = "center">
+<img src = "../images/stata-logfile.png" width="500" style="border:1px solid black;">
+<figcaption> Output if R script finds an error in the log</figcaption>
+</p>
+
+{{% /example %}}
+
+Remember, the above example assumes you've set up your Stata `do-file` to generate a log and you have the `logcheck.R` script as outlined before. Make sure to adjust the paths and filenames as per your specific project structure.
+
+## Additional Resources
```

Comment:
Nice touch with the example! 
Would be good to conclude with a separate "summary" section after additional resources..

Reply:
Perfect!

Summary added after additional resources ✅